libcloud/compute/drivers/brightbox.py

```
@@ -120,12 +121,15 @@ def _to_size(self, data):
         )
 
     def _to_location(self, data):
-        return NodeLocation(
-            id=data['id'],
-            name=data['handle'],
-            country='GB',
-            driver=self
-        )
+        if data:
```

I would get rid of this check - it just hides an actual error which is passing invalid data to this function (`None`, `''`, ...).